website/payments/services.py

```diff
@@ -8,22 +8,21 @@
 from django.utils import timezone, translation
 from django.utils.translation import gettext_lazy as _
 
-from members.models import Member
 from registrations.emails import _send_email
 from .exceptions import PaymentError
-from .models import Payment, BankAccount, Payable
+from .models import Payment, BankAccount, Payable, PaymentUser
 
 
 def create_payment(
     payable: Payable,
-    processed_by: Member,
+    processed_by: PaymentUser,
```

Comment:
Why does this need to be a PaymentUser exactly? You're proxying the model and the value here is saved immediately.